DigitalLearningSolutions.Web/Views/LearningPortal/Current/RemoveCurrentCourseConfirmation.cshtml

```diff
@@ -2,6 +2,13 @@
 @model CurrentCourseViewModel
 @{
   ViewData["Title"] = $"Learning Portal - {Model.Name} - Remove - Confirmation";
+  ViewData["Application"] = "Learning Portal - Remove Confirmation";

```

Comment:
Should just be learning portal 🙂

Reply:
Fixed for set completion date, remove confirmation and unlock in 689f1ae 🙂